specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -48,6 +48,12 @@ One example is features that were cut during scoping.
 
```

Comment:
Please elaborate on what frontend-ui is?

Reply:
I was going to do this as part of the data-pipelines PR. because that folder is more focused on the actual functionality. 
WDYT?

Reply:
you are mentioning the data-pipelines and the readme already, and it makes sense to either explain it now since mentioned, or move the data-pipelines-related changes entirely into a separate PR